src/MainBus.cpp

```diff
@@ -41,13 +41,34 @@ namespace sn
         }
         else if (addr < 0x6000)
         {
-            LOG(InfoVerbose) << "Expansion ROM read attempted. This is currently unsupported" << std::endl;
+            if (m_mapper->getType() == NROM474)
```

Comment:
Can we get rid of the type check and simply call m_mapper? That we leave it to the mapper and the memory bus code remains generic.